pyproject.toml

```diff
@@ -14,21 +14,21 @@ classifiers=[
     "License :: OSI Approved :: Apache Software License"
 ]
 dependencies = [
-    "caikit==0.7.0",
+    "caikit[all]>=0.9.0,<0.10.0",
     "caikit-tgis-backend>=0.1.6,<0.2.0",
 
     # TODO: loosen dependencies
     "accelerate>=0.18.0",
     "datasets>=2.4.0",
     "huggingface-hub",
-    "numpy==1.22.4",
-    "pandas==2.0.2",
-    "scikit-learn==1.2.2",
-    "scipy==1.8.1",
-    "tokenizers==0.13.3",
-    "torch==1.13.1",
-    "tqdm==4.65.0",
-    "transformers==4.27.1",
+    "numpy>=1.22.4",
+    "pandas>=1.5.0",
+    "scikit-learn>=1.1",
+    "scipy>=1.8.1",
+    "tokenizers>=0.13.3",
+    "torch>=1.13.1",
```

Comment:
This could have repercussion for TGIS, since different versions can cause side-effects